other-docs/guides/upgrading/v23.md

```diff
@@ -0,0 +1,115 @@
+modules to `^23.0.0`.
+
+```json
+{
+	"require": {
+		"altis/altis": "^23.0.0"
+	},
+	"require-dev": {
+		"altis/local-server": "^23.0.0"
+	},
+	"config": {
+		"platform": {
+			"php": "8.2"
+		}
+	}
+}
+```
+
+Once you have made these changes, run `composer update` and then run the `wp altis migrate` command:
+
+```sh
+# For cloud environments
+wp altis migrate
+
+# For local server
+composer server cli -- altis migrate
+```
+
+## WordPress 6.8
+
+Altis adds support to WordPress 6.8, "Cecil", which brings significant improvements to performance, security, and usability:
+
+- **Security**: bcrypt is now used for password hashing, replacing phpass. Application keys use stronger BLAKE2b hashing. Admins are alerted if open registration is active with high default roles.
+
+- **Performance**: Improved caching in WP_Query, better lazy loading, and speculative loading via the new Speculation Rules API for faster navigation.
+
+- **Editor Enhancements**: Updated Style Book UI, new Query Loop options (date filters, exclude sticky posts), and persistent rendering mode for editing experience.
+
+- **Accessibility**: Over 100 fixes across the admin and editor, including standardized tooltips and clearer admin notice prefixes.
+
+For more details check the [WordPress 6.8 release announcement](https://wordpress.org/download/releases/6-8/).
+
+## Headline Features
+
+### Altis Advanced Security, powered by Patchstack
+
+We're excited to launch Altis Advanced Security, a new additional add-on that provides real-time vulnerability protection for your Altis projects. Powered by Patchstack, this integration brings industry-leading virtual patching (vPatching) directly into your Altis environments.
+
+#### Why It Matters
+
+Identifying vulnerabilities is only the first step—mitigating them quickly is crucial. In many organizations, updating plugins or themes can take days or even weeks, leaving your sites exposed in the meantime.
+
+Even more critically, some plugins never release a fix, requiring manual workarounds or firewall rules.
+
+Altis Advanced Security solves this by automatically applying vPatches—plugin-level firewall rules that neutralize known vulnerabilities before an official fix is available. These rules are applied immediately and directly within WordPress, providing fast and targeted protection with minimal performance impact.
+
+#### Key Features
+
+- **Real-Time Vulnerability Mitigation**
+Instantly protects against known threats, without waiting for full plugin updates or release cycles.
+
+- **Smart vPatching**
+Only patches vulnerabilities in plugins you actually have installed, reducing overhead and improving efficiency.
+
+- **Seamless Integration**
+Works alongside Altis's built-in CDN-level WAF to provide protection across Layers 3, 4, and 7.
+
+- **Compliance-Friendly**
+Helps meet security expectations in compliance audits by proactively mitigating risks.
```

Comment:
These should be formatted with a separated column like below to make it more clear that it is a "title" and "subtitle"

**Real-Time Vulnerability Mitigation**: Instantly protects against known threats, without waiting for full plugin updates or release cycles.